docs/versioned_docs/version-zenith/user-guide/ci/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+slug: /user-guide/ci
+---
+
+import PartialExperimentalDocs from '../../partials/_experimental.mdx';
+
+# Dagger in CI
+
+<PartialExperimentalDocs />
+
+This guide explains how to integrate the Dagger SDKs with various CI services/tools.
```

Comment:
IMPORTANT: I have only tested the GHA version, the rest are guesstimates. As @kpenfound has the most experience with this, I would like him to review and update as needed.

Reply:
The CI configs look accurate to me